docs/quarantine.md

```diff
@@ -0,0 +1,95 @@
+behave deterministically until they are fixed, so that we can keep the rest of
+the suite as healthy as possible. You can read more about test quarantine
+methodology in [1] and [2], and about an actual implementation in [3].
+
+## Goal
+
+The purpose of appplying the methodology described in this document is increasing
+the stability of the CI suite. We consider the CI stable if the whole test suite
+has a failure rate below 10%.
+
+We need to multiplicate the individual failure rates to obtain the whole suite
+passing rate, this means that with a failure rate of 5% per individual test, more
+than 2 flaky tests would  lead to an overall test suite failure rate above the 10%
+goal (0.95 ** 2 = 0.9025). And, for instance, 17 failing tests with 5% failure rate
+would lead to a terrible 41.81% passing rate of the whole suite (0.95 ** 17 = 0.4181).
+
+## Procedure
+
+In order to remove as much as possible the influence of changes in PRs to
+determine the stability of the suite, we will take into account only results
+from the periodics that run e2e tests from master, these jobs can be checked
+[on testgrid].
+
+In the following definitions we will consider test failures only when reported on
+testgrid in jobs where less than 5 tests failed, so that we don't take into
+account systemic failures caused for instance by an infrastructure problem.
+
+### Putting tests in quarantine
+
+In the following failure rates are calculating taking into account periodics and
+presubmits that are executed on merged code (on tide merge batches).
+
+A test must be put in quarantine when any of these conditions is met:
+* It has a failure rate higher than 5% in the last two weeks.
+* It has a failure rate higher than 20% in the last 3 days.
+
+A PR will be proposed on Mondays every two weeks with a batch of the tests that
+met the first condition. A PR can be proposed at any time for the tests that meet
+the second condition. In both cases the PR will add the text `[QUARANTINE]` to
+each test's description in the code. An email will be sent to the owners of the
+suspected tests.
+
+After the PR with the quarantine candidates is proposed there is a grace period
+of 2 days to prepare and land a fix for a test in the batch. If at least 5
+consecutive executions with the fix pass the test can be removed from the batch.
+
+Each quarantined test must have a team owner. The PR will add the text
+`[owner:@sig-{compute,network,storage}]` to each test's description.
+
+When a test marked with the [release-blocker] meets the conditions to be
+quarantined we will:
+* Create github issue with a comment `/release-blocker master` to ensure that
+the issue is addressed before a new release is cut.
+* Ensure that the github issue is assigned to an individual who will own bringing
+the blocker to completion within a quick time frame.
+
+### Getting tests out of quarantine
+
+The person assigned to each tracking ticket should propose a fix for the test or,
```

Comment:
What is this "tracking ticket"? I do not think it was mentioned earlier, except for the blockers case which was talking about a github issue. But here it seems generic, so perhaps there is a need to specify when such an issue is to be opened.

Another point is about the owner: The text and in other places talked about an individual, but I would prefer to see here a group/SIG. As we mark ownership per SIG on the tests, I think we should do the same when talking about ownership. I guess we can mix this (SIG or person) if you think that level of granularity is needed.

Reply:
Yes, this "tracking ticket" is a leftover, removed. I agree also that the owner should be always the team, the "person" references also came from when we described management of tickets here, I've replaced all references person -> team.